content/docs/command-reference/exp/init.md

```diff
@@ -155,26 +149,23 @@ The easiest route is using interactive mode and answering a few questions:
 
 ```dvc
 $ dvc exp init --interactive
-This command will guide you to set up a train stage in dvc.yaml...
-
 Command to execute: python src/train.py
 
-Enter the paths for dependencies and outputs of the command.
-DVC assumes the following workspace structure:
-├── data
-├── metrics.json
-├── models
-├── params.yaml
-├── plots
-└── src
-
+Enter experiment dependencies.
 Path to a code file/directory [src, n to omit]: src/train.py
 Path to a data file/directory [data, n to omit]: data/features
-Path to a model file/directory [models, n to omit]: models/predict.h5
 Path to a parameters file [params.yaml, n to omit]:
+
+Enter experiment outputs.
+Path to a model file/directory [models, n to omit]: models/predict.h5
```

Comment:
This could be in a separate PR, but we should probably use `predict.h5` or `model.h5` instead of `models/predict.h5` until https://github.com/iterative/dvc/issues/5802 is resolved.

Reply:
Not sure. a) it's a bug I think (shouldn't doc based on that) and b) the output of `exp init` does say 

```
Ensure your experiment command creates ... models/predict.h5.
```

Reply:
Sorry, I should have clarified. This is what happens now:

```
$ dvc exp init -i
Command to execute: python src/train.py

Enter experiment dependencies.
Path to a code file/directory [src, n to omit]: src/train.py
'src/train.py' does not exist, the file will be created.
Path to a data file/directory [data, n to omit]: data/features
'data/features' does not exist, the directory will be created.
Path to a parameters file [params.yaml, n to omit]:
'params.yaml' does not exist, the file will be created.

Enter experiment outputs.
Path to a model file/directory [models, n to omit]: models/predict.h5
Path to a metrics file [metrics.json, n to omit]:
Path to a plots file/directory [plots, n to omit]: n

ERROR: unexpected error - [Errno 2] No such file or directory: '/private/tmp/tmprepo/models/.gitignore'

Having any troubles? Hit us up at https://dvc.org/support, we are always happy to help!
```

> it's a bug I think

Yeah, we need to prioritize fixing this in DVC.

Reply:
K. Marking this PR with `wait` status. Please lmk when to follow up if possible!